github/ipld.yml

```diff
@@ -3226,108 +3146,41 @@ teams:
       maintainer:
         - rvagg
         - SgtPooki
-      member:
-        - hacdias
-        - lidel
-        - olizilla
-        - whizzzkid
-    privacy: closed
```

Comment:
Maybe lets just remove the ipfs-gui team as a whole and inline to the one repo that uses this?